packages/datatrak-web-server/src/utils/formatTaskChanges.ts

```diff
@@ -28,13 +27,9 @@ export const formatTaskChanges = (task: Input) => {
     taskDetails.due_date = null;
   } else if (dueDate) {
     // apply status and due date only if not a repeating task
-    // set due date to end of day
-    const endOfDay = new Date(new Date(dueDate).setHours(23, 59, 59, 999));
+    const unix = new Date(dueDate).getTime();
```

Comment:
So it looks like every time we want to save a value to the due date col we will have to convert the date to the unix timestamp by calling getTime and then most times we want to read the value, we will have to convert it back using new Date(). I wonder if it would be possible to do that conversion at the model level so that we always get or send a date object or date iso string when working with the Task. In previous ORMs I've used that is a pretty common thing to do. @rohan-bes do you think there is a nice way to do this with our models?

Reply:
While this would be great, I don't think we have a pattern of doing this anywhere else. 
But, what I could do is, for fetching, add a custom column selector so that we don't have to convert this to a date object every time.

Reply:
yep that sounds like a good improvement to me 👍